### DISCUSSION TITLE
"TurboMode": false, breaks the mod?

### DISCUSSION THREAD

**Discussion by Escovas** [ssypchenko/cs2-gungame#13] [General]
hello I'm having a problem in my server, when changing the "TurboMode": to false in the "csgo\cfg\gungame.json" the gungame doesn't load correctly.
The plugin get loaded but the game mod itself not, if I type !rank, !top i get kicked from the server.

  "TurboMode": true,
  "_comment.TurboMode": "Turbo Mode: give next level weapon immediately on level up",

**Comment by ssypchenko** [comment]
> 
> "TurboMode": true, "_comment.TurboMode": "Turbo Mode: give next level weapon immediately on level up",

Hi, sorry for the long reply. I look at the code and I dont't see why this variable could affect the plugin in that way. 
I will check on my server in a couple of days, can't do it quicker. 

Just in case, what versions do you use: Metamod, CSS, GunGame?

**Reply by Escovas** [reply]
May  7 20:51:09:      Metamod:Source version 2.0.0-dev+1291

css Current API Version: v228 (1.0.228+11e5e99)

May  7 20:53:31:    [#2:LOADED]: "CS2_GunGame" (v1.1.0) by Sergey
May  7 20:53:31:      GunGame mode for CS2

1st I did make a config of my own and got that result, then I took the default config and changed option by option and the game went crazy after that cvar.

**Comment by Escovas** [comment]
I did recheck my server, and i dont know why but it works now.
probably i had something wrong in my config.
